### PR TITLE
Issue Verification and Deliverable #1 and #2

### PR DESCRIPTION
**Pull request by AgricolaKJ**

### CODE REVIEW

**Comment by AgricolaKJ** [comment]
The previous file uploads had the wrong dates as the file name.

**Comment by whanley** [comment]
Your deliverable 1 and 2 look really good. But the filename needs to be changed: it should be 1907-06-25.xml. Please fix it and send a new pull request.

Also, look at this tutorial for some tips: https://dig-eg-gaz.github.io/how-to/digitization/polished-tei-instructions/#1-get-the-page-hierarchy-right

File: 1905-08-28.xml [hunk 1]
```diff
@@ -1081,89 +1394,68 @@
                     <div type="item">
                         <head>PEACE PROSPECTS. Adjournment requested BY KAMURA. FRESH PROPOSALS
                             PROBABLE.</head>
-                        <p>Portsmouth (N.H.), August 26.</p>
-                        <p>There are no developments in the situation. The Russian attitude is
+                        <div type="wireReport"><p>Portsmouth (N.H.), August 26.There are no developments in the situation. The Russian attitude is
```
Comment (whanley) [inline 1905-08-28.xml]: please use `<dateline>`

File: 1905-08-28.xml [hunk 2]
```diff
@@ -1279,100 +1571,98 @@
...
-                <div type="section" feature="steamerMovements">
-                    <head>STEAMER MOVEMENTS.</head>
-                    <p>The S.S. Creole Prince left Malta with passengers and general cargo on
-                        Saturday, and is due here Wednesday next.</p>
-                    <p>The Messageries mail boat Senegal, which left Marseilles on the 84th inst,
-                        and passed the Straits of Messina yesterday, is due at Alexandria
-                        to-morrow.</p>
-                </div>
+                    <head>The Cutting of the Khalig.</head>
+                        <div type="item">
```
Comment (whanley) [inline 1905-08-28.xml]: don't put two divs inside this div--isn't it just one big item?

**Comment by whanley** [comment]
Good start on your deliverables. But why did you do 1 and 3 but skip 2 and 4 (the second half of the issues)?

Don't use `<div type="article">` -- use type="section" or "item".

There are some kinks to be ironed out, but mostly the work is pretty good.

Please look at your verification as well.

**Comment by AgricolaKJ** [comment]
Hi Dr. Hanley, I think I know what happened here with my deliverable two. I didn’t know much about  using the co tent folder to submit my assignments, so I saved my assignment in a different folder so that I could easily find it to submit it through the website. That could be the reason why you don’t have deliverable two. For deliverable #4, I hit the commit to master button on the Github desktop version. Did I miss a step when it came to submitting the last one?

Thanks again,
Kenneth Hills

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by whanley** [comment]
Let's take a look at the pull request to figure it out. We could do it after class tomorrow.

**Comment by AgricolaKJ** [comment]
Ok, thank you very much.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by AgricolaKJ** [comment]
Hi Dr.Hanley, I don’t see where to subunit the data visualization assignment

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by huoyanzhizhou** [comment]
Please remember to update the measures when you use advertisement/templates. For example, see 19070-07-27, line 326.

**Comment by whanley** [comment]
Great work--I'm going to merge these and see about completeness.

**Comment by whanley** [comment]
I'll call these complete, but please look back on them and see what you can clean further.

**Comment by AgricolaKJ** [comment]
Ok Dr.Hanley, thank you. I do have one question, is my issue verification done as well? I didn’t get a complete for it.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Pull request merged**